app/lib/katello/resources/candlepin.rb

```diff
@@ -65,9 +73,14 @@ def self.default_headers(uuid = nil)
             cp_oauth_header = User.cp_oauth_header
           end
 
-          {'accept' => 'application/json',
-           'accept-language' => I18n.locale,
-           'content-type' => 'application/json'}.merge(cp_oauth_header)
+          headers = {'accept' => 'application/json',
+                     'accept-language' => I18n.locale,
+                     'content-type' => 'application/json'}
+
+          request_id = ::Logging.mdc['request']
```

Comment:
What log is mdc?

Reply:
you can see it being set here:  https://github.com/theforeman/foreman/blob/698e916ce208b5040b83a908a058c83c94d158ee/lib/middleware/tagged_logging.rb

thats a special attribute on the logging to help it log the request id AFAIK